doc/changes/latest.inc

```diff
@@ -29,6 +29,7 @@ Enhancements
 Bugs
 ~~~~
 - Fix bug where channel names were not properly sanitized in :func:`mne.write_evokeds` and related functions (:gh:`11399` by `Eric Larson`_)
+- Fix bug where loading different combinations of volumes were in ``freeview`` when rotating and translating a volume to make a linear transformation array (LTA), different ``dst`` (destination) volumes were used in the LTA file, resulting in different affines for :func:`mne.read_lta` when the rotation and translations were the same (:gh:`11402` by `Alex Rockhill`_)
```

Comment:
I'm struggling to understand this sentence. Is this close? If not can you correct / improve it?

```suggestion
Fix bug where the wrong affine was returned when using :func:`mne.read_lta` to load Linear Transform Arrays (LTAs) made in freeview (:gh:`11402` by `Alex Rockhill`_)
```

Reply:
That's definitely more streamlined but I think it misses that the LTA was fine sometimes, it was just when someone loaded more volumes that it got messed up. I'll give it another pass.